src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+.main {
```

Comment:
Create a separate file for each BEM block

Reply:
Fixed